pkgs/misc/vscode-extensions/vscode-lldb/build-deps/package.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
It would help to include a script (maybe `generate.sh`?)  that would generate this automatically. Here's an example:

```bash
#!/usr/bin/env bash

# Script to generate package.json needed to create the build environment.

set -eu -o pipefail
cd "$(dirname "${BASH_SOURCE[0]}")"

SRC="$(nix-build -A vscode-extensions.vadimcn.vscode-lldb.src.src --no-out-link ../../../..)"
VERSION="$(nix-instantiate --eval -A vscode-extensions.vadimcn.vscode-lldb.version ../../../.. | tr -d '"')"
JQ_BIN="$(nix-build -A jq --no-out-link ../../../..)/bin/jq"


main() {
    jq "{\"name\": \"vscode-lldb\", \"version\": \"$VERSION\", \"dependencies\": (.devDependencies + .dependencies)}" \
        "$SRC/package.json" \
        > build-deps/package.json
}

jq() {
    command "$JQ_BIN" "$@"
}

main
```

Reply:
This PR has been going on for months, I think we could have another PR to do refinements.